example/cumulus-tf/lambdas.tf

```diff
@@ -52,3 +52,20 @@ resource "aws_lambda_function" "sns_s3_test" {
     security_group_ids = [aws_security_group.no_ingress_all_egress.id]
   }
 }
+
+resource "aws_lambda_function" "wait_for_s3_object_to_exist" {
```

Comment:
The previous workflow redeployment test used a task that waited for a CloudFormation update to finish. Since we are no longer using CloudFormation, that wouldn't work. Instead, I've added a task that waits for an S3 object to exist. The test now runs the deployment using Terraform and, when the deployment is finished, pushes an object to S3. The workflow sees that S3 object and completes.